bitcoin/podle.py

```diff
@@ -9,7 +10,7 @@
 from secp256k1_main import ctx
 PODLE_COMMIT_FILE = None
 N = 115792089237316195423570985008687907852837564279074904382605163141518161494337
-dummy_pub = secp256k1.PublicKey(ctx=ctx)
+dummy_pub = coincurve.PublicKey(None)
```

Comment:
It seems like dummy_pub is only ever used for combine() which never accesses self.public_key. Could be useful to have combine as a classmethod if it doesn't break anything?